### PR TITLE
monorepo: add angular support to scripts

### PR DESCRIPTION
**Pull request by christianvogt**
Tested by building the angular tour of heroes app.

Copied settings from the `fabric8-ui` webpack configs into the `scripts` package. No AOT support yet but at least I was able to build and run the dev server for the angular tour of heroes app.

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @christianvogt for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin.

File: talamer/package.json [hunk 1]
```diff
@@ -29,15 +29,31 @@
...
+    "react": "16.5.2",
+    "react-dom": "16.5.2",
+    "react-router": "4.3.1",
+    "react-router-dom": "4.3.1",
+    "rxjs": "6.3.3",
+    "rxjs-compat": "6.3.3",
+    "zone.js": "0.8.26"
   },
   "devDependencies": {
+    "@angular/compiler-cli": "^6.8.8",
```
Comment (joshuawilson) [inline talamer/package.json]: Is this really `6.8.8` and not `6.1.8`

Comment (christianvogt) [reply talamer/package.json]: I'll fix this.

**Comment by alien-ike** [comment]
Hey @christianvogt! It seems you tried to trigger `/ok-without-tests` command, but this will not have any effect due to insufficient permission. You have to be admin or requested reviewer or pull request approver, but not pull request creator for this command to take an effect.

**Pull request merged**